Redo advanced settings

ulisesmac opened the pull request:
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
fixes #21896 
fixes #21360

## Summary
Advanced settings + legacy settings reworked.

An improvement on a better explanation on what exactly each setting does is left for future work with less priority.

Demo:

https://github.com/user-attachments/assets/b0a1c234-7491-4049-8b56-3e3b501be831

Additional UI fixes:

Before/After(gap fix, now it matches Figma designs):
<img src="https://github.com/user-attachments/assets/b64a7dc0-0c74-4bc1-a830-c0fa929d813a" width="300"> <img src="https://github.com/user-attachments/assets/f938f00c-c4d6-4e97-893b-99dd15944e0f" width="300">

Before/After (Icon colors fixed to match Figma):

<img src="https://github.com/user-attachments/assets/b64a7dc0-0c74-4bc1-a830-c0fa929d813a" width="300"> <img src="https://github.com/user-attachments/assets/58d65981-cf65-46b2-b8fc-39305c5c1b43" width="300">


## Review notes

This is also part of what is expected:
https://discord.com/channels/1210237582470807632/1364364039752384644/1364448582245290095

### Platforms

- Android
- iOS

status: ready

ulisesmac committed:
Fix gaps in options

ulisesmac committed:
Fix icon colors in profile page

ulisesmac committed:
Pass customization color to settings-item's selector component

ulisesmac committed:
Spread blur property across category-settings children so that the UI is consistent

ulisesmac committed:
Pass `on-press` callback to selector inside drawer-action

ulisesmac committed:
Add new advanced settings screen

ulisesmac committed:
Remove legacy advanced settings screen

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (8)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | ac586981 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22576/1/) | 2025-05-13 03:26:25 | ~5 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250513-032105-ac5869-pr22576-tests.log) |
| :heavy_check_mark: | ac586981 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22576/1/) | 2025-05-13 03:31:14 | ~10 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250513-032100-ac5869-pr22576-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250513-032100-ac5869-pr22576-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | ac586981 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22576/1/) | 2025-05-13 03:31:52 | ~10 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250513-032100-ac5869-pr22576-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250513-032100-ac5869-pr22576-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | ac586981 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22576/1/) | 2025-05-13 03:33:25 | ~12 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/tjZdBN) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FtjZdBN&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :heavy_check_mark: | 3a0cae2c | [#2](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22576/2/) | 2025-05-13 10:03:08 | ~5 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250513-095801-3a0cae-pr22576-tests.log) |
| :heavy_check_mark: | 3a0cae2c | [#2](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22576/2/) | 2025-05-13 10:06:47 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250513-095756-3a0cae-pr22576-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250513-095756-3a0cae-pr22576-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 3a0cae2c | [#2](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22576/2/) | 2025-05-13 10:07:12 | ~9 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250513-095756-3a0cae-pr22576-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250513-095756-3a0cae-pr22576-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 3a0cae2c | [#2](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22576/2/) | 2025-05-13 10:12:31 | ~14 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/pi3M9x) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2Fpi3M9x&size=400x400&errorCorrectionLevel=L)|
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 56c8b1c3 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22576/4/) | 2025-05-14 01:34:41 | ~7 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250514-012708-56c8b1-pr22576-tests.log) |
| :heavy_check_mark: | 56c8b1c3 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22576/4/) | 2025-05-14 01:37:54 | ~10 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250514-012704-56c8b1-pr22576-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250514-012704-56c8b1-pr22576-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 56c8b1c3 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22576/4/) | 2025-05-14 01:38:36 | ~11 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250514-012708-56c8b1-pr22576-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250514-012708-56c8b1-pr22576-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 56c8b1c3 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22576/4/) | 2025-05-14 01:39:38 | ~12 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/Z4r3RM) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FZ4r3RM&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :heavy_check_mark: | 81358fe7 | [#5](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22576/5/) | 2025-05-14 23:27:29 | ~5 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250514-232223-81358f-pr22576-tests.log) |
| :heavy_check_mark: | 81358fe7 | [#5](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22576/5/) | 2025-05-14 23:32:17 | ~9 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250514-232218-81358f-pr22576-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250514-232218-81358f-pr22576-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 81358fe7 | [#5](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22576/5/) | 2025-05-14 23:32:51 | ~10 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250514-232223-81358f-pr22576-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250514-232223-81358f-pr22576-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 81358fe7 | [#5](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22576/5/) | 2025-05-14 23:34:32 | ~12 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/5KznF3) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2F5KznF3&size=400x400&errorCorrectionLevel=L)|

Parveshdhull commented on src/quo/components/drawers/drawer_action/view.cljs:
```diff
@@ -67,6 +67,7 @@
           :label-prefix        "toggle"
           :customization-color customization-color
           :type                :toggle
+          :on-change           on-press
```
Hi @ulisesmac, Thank you very much your great work.

Please can you elaborate this change?

ulisesmac replied:
Sure @Parveshdhull !

This is the component:
![image](https://github.com/user-attachments/assets/a0c10e91-3bd1-4f12-a104-bf5d1bad5e81)

The problem was the actual selector didn't trigger the `on-press` callback, which is a bit weird, it was only triggered when pressed outside, from a UX perspective it was even confusing

Parveshdhull commented on translations/en.json:
```diff
@@ -809,6 +809,9 @@
   "disable-later-in-settings": "You can disable this later in Settings",
+  "disable-light-client": "Disable Light Client?\n\nYou’ll need to log in again.",
+  "disable-store-confirmations": "Disable Store confirmations?\n\nYou’ll need to log in again.",
+
```
empty line

Parveshdhull commented on translations/en.json:
```diff
@@ -908,10 +911,12 @@
   "enable-all": "Enable all",
   "enable-biometrics": "Enable biometrics",
   "enable-camera": "Enable camera",
+  "enable-light-client": "Enable Light Client?\n\nYou’ll need to log in again.",
```
Should we create a different entry for 'You’ll need to log in again'? I'm not sure how we will support multiple languages, but since this line is used multiple times, having a separate entry might save some time.

flexsurfer replied:
yeah \n\n shouldn't be used

ulisesmac replied:
This will appear on a `js/alert` and these settings are mostly used for us (devs) and QA. Also a better explanation on every setting will be done as a low prio PR.

Just used `\n\n` to set a proper spacing on the alerts, we cannot put two separate strings on the alerts. I think it's fine as is unless there's another argument to avoid it.

Parveshdhull commented on src/status_im/contexts/settings/advanced/events.cljs:
```diff
@@ -0,0 +1,120 @@
+            (assoc-in [:profile/profile :last-backup] last-backup))}))
+
+(rf/reg-event-fx
+ ::backup-failed
+ (fn [{:keys [db]}]
```
Why not `:advanced-settings/backup-failed`?

ulisesmac replied:
I'm inheriting the previous implementation, and actually it's a great idea.

Why not? because this is a "private" event, it shouldn't be accessible from outside, this way we are not easily exposing this event.

ilmotta replied:
It's not clear that the namespaced keyword means "private". Doesn't seem like a future-proof choice as well as we might prefer to use namespaced keys for events in the future. Better seems to go with our current standards and consider a suggestion for consistently naming private events that should only be called from other events in the same namespace.

ulisesmac replied:
Ok, changed.

Parveshdhull commented on src/status_im/contexts/settings/advanced/view.cljs:
```diff
@@ -0,0 +1,259 @@
+        performing-backup?    (rf/sub [:backup/performing-backup])
+        customization-color   (rf/sub [:profile/customization-color])]
+    [:<>
+     [quo/drawer-top {:title "Waku Backup"}]
```
This and other occurrences in the file can be migrated to translations

ulisesmac replied:
Yes, I forgot to move some strings to translations

Parveshdhull commented on src/status_im/contexts/settings/advanced/view.cljs:
```diff
@@ -0,0 +1,259 @@
+        :accessibility-label :waku-backup-switch
+        :icon                :i/revert
+        :customization-color customization-color
+        :on-press            #(rf/dispatch [:advanced-settings/toggle-waku-backup])}]]
```
Should we extract these functions separetly, or use rn/callback?

ulisesmac replied:
It's not needed, the cost is extremely cheap.

flexsurfer commented on src/status_im/common/signals/events.cljs:
```diff
@@ -108,6 +108,9 @@
 
       "backup.performed"
-      {:db (assoc-in db [:profile/profile :last-backup] (oops/oget event-js :lastBackup))}
+      ;; Delay to improve the UX, if no delay is added, messages appear/disappear too fast.
```
could you pls elaborate more what's the problem in details

ulisesmac replied:
We have a message in the UI saying "Performing backup" but the signal returns too fast, so we dispatch the success event a little delayed so the UX is better, if it happens too fast the user isn't able to read the message.

flexsurfer replied:
shouldn't this delay be in the component instead? because it might happen in other cases as well

ilmotta replied:
Going slightly off-topic: interestingly, again what we sometimes need in re-frame is to have a cleaner control of a priority queue. When this `backup.performed` signal arrives, it's not important to process it immediately and interrupt other important computations. There could be an effect named like:

```clojure
:fx [[:dispatch-x {:priority :low :dispatch event-vector ...
```

ulisesmac replied:
@flexsurfer I think it's harder.

Also probably all these waku related settings will be changed/reviewed later when Waku team revisits the API they provide us.

flexsurfer commented on src/status_im/contexts/settings/advanced/view.cljs:
```diff
@@ -0,0 +1,259 @@
+        current-fleet (rf/sub [:fleets/current-fleet])
+        options
+        (rn/use-memo
+         (fn []
```
probably better to move it to defined function

ulisesmac replied:
I didn't notice `lint-fix` moved it like this, I'll refactor it :+1:

ulisesmac committed:
Extract options to separate function

ulisesmac committed:
Fix code style

ulisesmac committed:
Fix translations

ulisesmac committed:
lint fix

ulisesmac commented:
@flexsurfer @Parveshdhull  thanks for the review!, comments solved

flexsurfer reviewed (approved):
thanks

ilmotta commented on src/status_im/contexts/settings/advanced/view.cljs:
```diff
@@ -0,0 +1,266 @@
+        customization-color   (rf/sub [:profile/customization-color])]
+    [:<>
+     [quo/drawer-top {:title (i18n/label :t/waku-backup)}]
+     (comment
```
Is this a leftover?

ulisesmac replied:
Nope, this code is useful to complete the UI with an explanation on what enabling the backup implies. I'd prefer to keep it and just adjust it in a follow up where all the settings can provide an explanation on what they do

ilmotta replied:
If you are sure you'll be able to work on the follow-up soon(ish) that's okay, otherwise it'd be a good to explain this as a TODO, or even better have an issue describing this follow-up work to help us more easily manage the project.

ulisesmac replied:
I'll open an issue @ilmotta

ilmotta reviewed (approved):
Code looks much nicer and great to see these changes getting in the app. Thanks @ulisesmac

ulisesmac committed:
Rename events with `::`

status-im-auto commented:
## 96% of end-end tests have passed
```
Total executed tests: 24
Failed tests: 1
Expected to fail tests: 0
Passed tests: 23
```
```
IDs of failed tests: 741612 
```
<h3>Failed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=redo-advanced-settings&PR_ID=22576&APK_NAME=22576.apk&TR_CASE_IDS=741612">Rerun failed tests</a></li><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859547">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
wallet/test_wallet_mainnet_no_send.py:345: in test_wallet_bridge_flow_mainnet
    self.wallet_view.slide_button_track.slide()
../views/base_view.py:107: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:303: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:289: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076312855641DPZ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (23)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859552">test_profile_change_profile_photo</a>, id: 741969 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076307856374AFB">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076441874918DWK">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859551">test_profile_change_username</a>, id: 741968 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076307856374AFB">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076441874918DWK">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCustomParamOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859555">test_send_snt_custom_tx_params</a>, id: 742910 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186232304344473BUA">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859540">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076474743705YVC">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076396245359PRJ">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859533">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076474743705YVC">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076396245359PRJ">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859532">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076474743705YVC">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076396245359PRJ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDeviceTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859543">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076445289142THM">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859546">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076312855641DPZ">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859544">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076312855641DPZ">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859545">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076312855641DPZ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859549">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076434182921SSP">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859548">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076434182921SSP">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859550">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076434182921SSP">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859535">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076447500558TJZ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076417816141KIR">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076470069630ROU">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859537">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076420822123YEV">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076650547646IRG">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859536">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076420822123YEV">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076650547646IRG">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859539">test_community_one_image_send_reply_set_reaction</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076420822123YEV">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076650547646IRG">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859538">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076420822123YEV">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076650547646IRG">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859541">test_send_eth[Status Network Sepolia-0.0002]</a>, id: 727229 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076420176182TOL">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076436839315KDR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859542">test_wallet_send_erc20_from_drawer[Optimism Sepolia-USD Coin-USDC-2-0.01]</a>, id: 727230 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076420176182TOL">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076436839315KDR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859553">test_send_eth[Arbitrum Sepolia-0.0001]</a>, id: 742015 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076420176182TOL">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076436839315KDR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859554">test_wallet_send_erc20_from_drawer[Sepolia-USD Coin-USDC-2-0.01]</a>, id: 742016 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076420176182TOL">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076436839315KDR">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3859534">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747186076406197724PFO">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

VolodLytvynenko commented:
Hi @ulisesmac thank you for PR. Take a look at the found issue

## PR_ISSUE 1: No navigation to relogin after fleet is changed

### Steps:
1. Go to advances settings
2. Change fleet -> confirm fleet changes

### Actual result:
After confirming the fleet change, the user is not navigated to the relogin flow.

https://github.com/user-attachments/assets/71e2eefa-e960-4b3a-92cd-d1e3510a4ec2

### Expected result:
After the fleet change is confirmed the user should be navigated to the relogin flow.

### OS: 
IOS, Android


### Devices:
- Pixel 7a, Android 15
- iPhone 11 Pro Max, IOS 17
### Logs
https://drive.google.com/file/d/1SnvqL3X9JkecEttq6GCDbkB4m3HLfagB/view?usp=drive_link

ulisesmac committed:
Fix bug when logging out

ulisesmac commented:
@VolodLytvynenko Thanks for testing!

I fixed the issue, there was also an issue related about the screen not being interactable when the logout happened (now fixed).

Can you please test again?

ulisesmac commented:
@ilmotta Here's the follow up to add more detail on the settings:
- https://github.com/status-im/status-mobile/issues/22585

@VolodLytvynenko Please consider the description of each setting needs to be improved and that should be solved in a following PR (low prio)

status-im-auto commented:
## 79% of end-end tests have passed
```
Total executed tests: 24
Failed tests: 5
Expected to fail tests: 0
Passed tests: 19
```
```
IDs of failed tests: 727230,741612,741841,741968,742016 
```
<h3>Failed tests (5)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=redo-advanced-settings&PR_ID=22576&APK_NAME=22576.apk&TR_CASE_IDS=727230,741612,741841,741968,742016">Rerun failed tests</a></li><br/><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860436">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `CollectibleItemElement` by `xpath`: `//*[@content-desc='collectible-list-item']//*[contains(@text,'Glitch Punks')]/../..` </div><div>Device 1: Find `CollectibleItemElement` by `xpath`: `//*[@content-desc='collectible-list-item']//*[contains(@text,'Glitch Punks')]/../..` </div></blockquote></p>

```
wallet/test_collectibles.py:159: in test_wallet_collectible_send_from_expanded_info_view
    self.wallet_view.get_collectible_element('Glitch Punks').scroll_and_click()
../views/base_element.py:209: in scroll_and_click
    self.scroll_to_element(direction=direction)
../views/base_element.py:204: in scroll_to_element
    raise NoSuchElementException(
 Device 1: CollectibleItemElement by xpath: `//*[@content-desc='collectible-list-item']//*[contains(@text,'Glitch Punks')]/../..` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440059721688BVM">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860428">test_wallet_send_erc20_from_drawer[Optimism Sepolia-USD Coin-USDC-2-0.01]</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='container']/android.widget.TextView[@text='USD Coin']/../android.widget.TextView[3]` </div><div>Device 1: `Text` is `47.24 USDC (EVM)` </div></blockquote></p>

```
wallet_txs/test_wallet_testnet.py:101: in test_wallet_send_erc20_from_drawer
    self.errors.verify_no_errors()
base_test_case.py:207: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: Verification of last transaction failed.The failed check: amount (Expected: 0.01 USDC, Found: 0.01 USDC (EVM))
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440026982876AXH">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439820077880OZO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860440">test_wallet_send_erc20_from_drawer[Sepolia-USD Coin-USDC-2-0.01]</a>, id: 742016 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='container']/android.widget.TextView[@text='USD Coin']/../android.widget.TextView[3]` </div><div>Device 1: `Text` is `47.25 USDC (EVM)` </div></blockquote></p>

```
wallet_txs/test_wallet_testnet.py:101: in test_wallet_send_erc20_from_drawer
    self.errors.verify_no_errors()
base_test_case.py:207: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: Verification of last transaction failed.The failed check: amount (Expected: 0.01 USDC, Found: 0.01 USDC (EVM))
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440026982876AXH">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439820077880OZO">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860437">test_profile_change_username</a>, id: 741968 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: LogInButton </div><div>## Signed in successfully!</div></blockquote></p>

```
profile/test_profile.py:180: in test_profile_change_username
    self.errors.verify_no_errors()
base_test_case.py:207: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Updated username 'user 1' is not shown in the community channel
E    Device 2: Updated username is not shown in the mentions list
E    Device 2: Updated username is absent in the contacts list
E    Device 2: Can't find chat with updated username
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439804910666SFK">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440076937240MGF">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860433">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
wallet/test_wallet_mainnet_no_send.py:345: in test_wallet_bridge_flow_mainnet
    self.wallet_view.slide_button_track.slide()
../views/base_view.py:107: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:303: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:289: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439820651464SGC">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (19)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDeviceTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860429">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439910354081GWH">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860421">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440205817534BDB">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440047900162DNQ">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439802481144MOG">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860427">test_send_eth[Status Network Sepolia-0.0002]</a>, id: 727229 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440026982876AXH">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439820077880OZO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860439">test_send_eth[Arbitrum Sepolia-0.0001]</a>, id: 742015 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440026982876AXH">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439820077880OZO">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860438">test_profile_change_profile_photo</a>, id: 741969 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439804910666SFK">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440076937240MGF">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860423">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440071301990UZA">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440219688516MEK">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860422">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440071301990UZA">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440219688516MEK">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860425">test_community_one_image_send_reply_set_reaction</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440071301990UZA">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440219688516MEK">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860424">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440071301990UZA">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440219688516MEK">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCustomParamOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860441">test_send_snt_custom_tx_params</a>, id: 742910 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308557413224255HAS">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860426">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440201721730YCD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439915740412HAC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860419">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440201721730YCD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439915740412HAC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860418">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440201721730YCD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439915740412HAC">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860435">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440059721688BVM">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860434">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308440059721688BVM">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860432">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439820651464SGC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860430">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439820651464SGC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860431">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439820651464SGC">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860420">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747308439865086909DVA">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

status-im-auto commented:
## 20% of end-end tests have passed
```
Total executed tests: 5
Failed tests: 4
Expected to fail tests: 0
Passed tests: 1
```
```
IDs of failed tests: 727230,741612,741968,742016 
```
<h3>Failed tests (4)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=redo-advanced-settings&PR_ID=22576&APK_NAME=22576.apk&TR_CASE_IDS=727230,741612,741968,742016">Rerun failed tests</a></li><br/><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860457">test_profile_change_username</a>, id: 741968 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: LogInButton </div><div>## Signed in successfully!</div></blockquote></p>

```
profile/test_profile.py:180: in test_profile_change_username
    self.errors.verify_no_errors()
base_test_case.py:207: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Updated username 'user 1' is not shown in the community channel
E    Device 2: Updated username is not shown in the mentions list
E    Device 2: Updated username is absent in the contacts list
E    Device 2: Can't find chat with updated username
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747311251397823027ZPR">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747311251412410806KML">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860454">test_wallet_send_erc20_from_drawer[Optimism Sepolia-USD Coin-USDC-2-0.01]</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='container']/android.widget.TextView[@text='USD Coin']/../android.widget.TextView[3]` </div><div>Device 1: `Text` is `47.25 USDC (EVM)` </div></blockquote></p>

```
wallet_txs/test_wallet_testnet.py:101: in test_wallet_send_erc20_from_drawer
    self.errors.verify_no_errors()
base_test_case.py:207: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: Verification of last transaction failed.The failed check: amount (Expected: 0.01 USDC, Found: 0.01 USDC (EVM))
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747311251374052643TWT">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747311251469071400EVR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860458">test_wallet_send_erc20_from_drawer[Sepolia-USD Coin-USDC-2-0.01]</a>, id: 742016 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='container']/android.widget.TextView[@text='USD Coin']/../android.widget.TextView[3]` </div><div>Device 1: `Text` is `47.26 USDC (EVM)` </div></blockquote></p>

```
wallet_txs/test_wallet_testnet.py:101: in test_wallet_send_erc20_from_drawer
    self.errors.verify_no_errors()
base_test_case.py:207: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: Verification of last transaction failed.The failed check: amount (Expected: 0.01 USDC, Found: 0.01 USDC (EVM))
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747311251374052643TWT">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747311251469071400EVR">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860455">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div><div>Device 1: Click system back button </div></blockquote></p>

```
wallet/test_wallet_mainnet_no_send.py:349: in test_wallet_bridge_flow_mainnet
    self.errors.verify_no_errors()
base_test_case.py:207: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: Base to Optimism: Max fees is not shown before pressing Review Bridge button
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747311251373637226JBL">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3860456">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=NAAV-AND-2130125-1747311251454671940HKQ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

VolodLytvynenko commented:
Hi @ulisesmac thank you for the fixes. No issues from my side. PR can be merged
